configure

```diff
@@ -769,6 +769,8 @@ def configure_node(o):
   if options.enable_static:
     o['variables']['node_target_type'] = 'static_library'
 
+  o['variables']['node_module_version'] = 46
```

Comment:
Should probable be added to release docs:

https://github.com/nodejs/node/blob/master/doc/releases.md#2-update-srcnode_versionh